client/ayon_nuke/plugins/load/load_clip.py

```diff
@@ -64,13 +64,25 @@ class LoadClip(plugin.NukeLoader):
     options_defaults = {
         "start_at_workfile": True,
         "add_retime": True,
-        "deep_exr": False
+        "node_type": "auto",
     }
 
     node_name_template = "{class_name}_{ext}"
 
     @classmethod
     def get_options(cls, *args):
+
+        # map default value (str) to index
+        # "newer" versions of qargparse support providing default value as string
+        # see https://github.com/mottosso/qargparse.py/pull/10
```

Comment:
~~@iLLiCiTiT didn't we stop using `qargparse` anyway?~~

I see this code is still using `qargparse`. It may be good to update anyway and replace them with AYON Attribute Defs which we also support as options here.